docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+across cluster-wide resources. With ClusterResourceOverride, you can define rules based on cluster labels or other 
+criteria, specifying changes to be applied to various cluster-wide resources such as namespaces, roles, role bindings, 
+or custom resource definitions. These modifications may include updates to permissions, configurations, or other 
+parameters, ensuring consistent management and enforcement of configurations across your Fleet-managed Kubernetes clusters.
+
+## API Components
+The ResourceOverride API consists of the following components:
+- **Cluster Resource Selectors**: These specify the set of cluster resources selected for overriding.
+- **Policy**: This specifies the policy to be applied to the selected resources.
+
+
+The following sections discuss these components in depth.
+
+## Cluster Resource Selectors
+A `ClusterResourceOverride` object may feature one or more cluster resource selectors, specifying which resources to select to be overridden.
+
+The `ClusterResourceSelector` object supports the following fields:
+- `group`: The API group of the resource
+- `version`: The API version of the resource
+- `kind`: The kind of the resource
+- `name`: The name of the resource
+> Note: The resource can only be selected by name.
+
+
+To add a resource selector, edit the `clusterResourceSelectors` field in the `ClusterResourceOverride` spec:
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: example-cro
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+```
+
+The example above will pick a `ClusterRole` named `secret-reader` to be overridden.
+
+
+## Policy
+The `Policy` is made up of a set of rules (`OverrideRules`) that specify the changes to be applied to the selected
+resources on selected clusters.
+
+Each `OverrideRule` supports the following fields:
+- **Cluster Selector**: This specifies the set of clusters to which the override applies.
+- **JSON Patch Override**: This specifies the changes to be applied to the selected resources.
+
+To add an override rule, edit the `policy` field in the `ClusterResourceOverride` spec:
+
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: example-cro
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+```
+The `ClusterResourceOverride` object above will remove the verb "list" in the `ClusterRole` named `secret-reader` on 

```

Comment:
Small nit. How do we know it is verb "list" that is removed in this example from path: /rules/0/verbs/2?